src/vcpkg/commands.add-version.cpp

```diff
@@ -326,35 +423,31 @@ namespace vcpkg::Commands::AddVersion
         const bool skip_formatting_check = Util::Sets::contains(parsed_args.switches, OPTION_SKIP_FORMATTING_CHECK);
         const bool skip_version_format_check =
             Util::Sets::contains(parsed_args.switches, OPTION_SKIP_VERSION_FORMAT_CHECK);
-        const bool verbose = Util::Sets::contains(parsed_args.switches, OPTION_VERBOSE);
+        const bool verbose = !add_all || Util::Sets::contains(parsed_args.switches, OPTION_VERBOSE);
 
         auto& fs = paths.get_filesystem();
         auto baseline_path = paths.builtin_registry_versions / "baseline.json";
         if (!fs.exists(baseline_path, VCPKG_LINE_INFO))
         {
-            vcpkg::printf(Color::error, "Error: Couldn't find required file `%s`\n.", baseline_path);
-            Checks::exit_fail(VCPKG_LINE_INFO);
+            Checks::msg_exit_with_error(
+                VCPKG_LINE_INFO, msgAddVersionFileNotFound, msg::path = baseline_path.generic_u8string());
         }
 
         std::vector<std::string> port_names;
         if (!args.command_arguments.empty())
         {
             if (add_all)
             {
-                vcpkg::printf(Color::warning,
-                              "Warning: Ignoring option `--%s` since a port name argument was provided.\n",
-                              OPTION_ALL);
+                msg::print_warning(msgAddVersionIgnoringOptionAll, msg::option = Strings::concat("--", OPTION_ALL));
```

Comment:
The current message convention is to include the `--` in the message template itself, and pass the option directly to `msg::option`.